pyboy/core/cartridge/base_mbc.py

```diff
@@ -103,7 +105,55 @@ def init_rambanks(self, n):
         self.rambanks = memoryview(array.array("B", [0] * (8*1024*16))).cast("B", shape=(16, 8 * 1024))
 
     def getgamename(self, rombanks):
-        return "".join([chr(rombanks[0, x]) for x in range(0x0134, 0x0142)]).split("\0")[0]
+        return "".join([chr(rombanks[0, x]) for x in range(0x0134, 0x0143)]).split("\0")[0]
+
+    def getgametype(self, rombanks):
+        if rombanks[0, 0x143] == 0x80 or rombanks[0, 0x143] == 0xc0:
+            return "Game Boy Color"
+        elif rombanks[0, 0x146] == 0x03:
+            return "Super Game Boy"
+        else:
+            return "Game Boy"
+
+    def getgameregion(self, rombanks):
+        game_type = self.getgametype(rombanks)
+        # The region character is only present in GBC/SGB games only; for regular GB games can't be fetched
+        if game_type == 'Game Boy':
+            return "Unknown"
```

Comment:
It would be better to return `None`, when we don't have an answer. It's more intuitive that it returns something that evaluates to False: `bool(pyboy.cartridge_region) == False`.

I.e. this won't work if it's `"Unknown"`:
```python
if python.cartridge_region:
    do_something(python.cartridge_region)
```